src/common/src/system_param/mod.rs

```diff
@@ -87,7 +87,10 @@ macro_rules! for_all_params {
             { max_concurrent_creating_streaming_jobs,   u32,    Some(1_u32),                                true,   "Max number of concurrent creating streaming jobs.", },
             { pause_on_next_bootstrap,                  bool,   Some(false),                                true,   "Whether to pause all data sources on next bootstrap.", },
             { enable_tracing,                           bool,   Some(false),                                true,   "Whether to enable distributed tracing.", },
-            { use_new_object_prefix_strategy,           bool,   None,                                       false,   "Whether to split object prefix.", },
+            { use_new_object_prefix_strategy,           bool,   None,                                       false,  "Whether to split object prefix.", },
+            { minitrace,                                bool,   Some(false),                                true,   "Whether to enable minitrace tracing.", },
```

Comment:
Can we also reuse the existing `enable_tracing` for tiered cache tracing?

Reply:
LGTM. I'll update it with the endpoint configuration.